### ISSUE TITLE
Uneven Unicode Support when "Writing" Text

### ISSUE THREAD

**Issue by zefrawg** [timothycrosley/streamdeck-ui#240]
I am opening this here because I cannot reopen [#85](https://github.com/timothycrosley/streamdeck-ui/issues/85) and I'm not entirely sure it's the same issue.

**Describe the bug**
When pressing a Stream Deck button configured to 'write' a Unicode character with codepoint U+16D7 or higher, the button will 'write' only the text before the first such codepoint and then stop output with a console message:

`Could not complete the write command: (6, 'ᛝ')`

(This does not crash the program or seem to affect other functionality.)

**To Reproduce**
1. Configure a Stream Deck button to write "Hello ᛝ World" on click.
2. While in a text editor, press the configured button.
3. Note that only "Hello " appears in the text editor.
4. Review console output, noting the error message as indicated above.

**Expected behavior**
The message "Hello ᛝ World" appears in the text editor.

**Operating system (please complete the following information)**
 - OS: Ubuntu 22.04
 - Desktop environment: Gnome
 - Python version [3.10]

**Stream Deck Version**
 - XL

**Comment by zefrawg** [comment]
Update: The problem buttons started working seemingly spontaneously. (Restarting the software had not fixed the problem.) I have rebooted since the problem began, but I can't be sure that's why or when the buttons started working.

However, since it was working I started adding a new page of similar buttons and the same problem occurred again and also eventually resolved itself again.

Both times I was working specifically with the Runic unicode block, however as noted above, I tried others as well with the same result.

Unfortunately, and unhelpfully, I'm not actually sure what's triggering or fixing the problem at this point.
